packages/core/usePictureInPicture/index.ts

```diff
@@ -0,0 +1,43 @@
+import { computed, ref } from 'vue-demi'
+
+import type { MaybeElementRef } from '../unrefElement'
+
+import { useEventListener } from '../useEventListener'
+
+export function usePictureInPicture(
+  el: MaybeElementRef<HTMLVideoElement | undefined>,
+) {
+  const video = ref(el)
+
+  const isSupported = computed(() => {
```

Comment:
use `useSupported` instead and maybe we should add document && .. for friendly SSR support

Reply:
Done ✅